test/algorithms/canny.jl

```diff
@@ -149,10 +155,10 @@
     end
 
     @testset "Default Values" begin
-        img = Gray{N0f8}.(load("algorithms/References/circle.png"))
+        img = Gray{N0f8}.(testimage("cameraman"))
         out, offsets = detect_subpixel_edges(img)
-        @test_reference "References/circle_edge.png" Gray.(detect_edges(img)) by=edge_detection_equality()
```

Comment:
do we need to remove `circle_edge.png`?

FYI, one way to quickly rebuild all the references is to remove the whole `References` folder.

Reply:
For this particular test we unfortunately need to remove it. The issue is that by using the KernelFactors.gaussian() instead of KernelFactors.IIRGaussian() the resulting image is different in a subtle way. The image has fewer unique intensity values. This subtle changes messes up the implicit threshold that is associated with the "Default Values" of `low =  Percentile(20)` and `high = Percentile(80)`. Whereas these percentiles gave the correct result and were good default values for this particular edge image when we used the IIRGaussian filter, they are a poor choice for this particular edge image. A suitable threshold fior this image would be `low =  Percentile(75)` and `high = Percentile(80)`. The trouble is that this test is specifically meant to execute the API for the default values. Hence, I decided to go with a different image altogether because it doesn't make sense to change the default values to `low =  Percentile(75)` and `high = Percentile(80)` in the general case.

Reply:
Oh sorry, I didn't notice this `circle_edge.png` is used in L108.

BTW, it seems that you've accidentally committed two `cameraman_edge.png` files.

Reply:
I might just add that this was a very annoying and perplexing discovery. :)

Reply:
and yet some bad luck in the nightly version :(